src/app/components/charts/LineChart.tsx

```diff
@@ -57,6 +60,9 @@ const LineChartCmp = <T extends object>({
         tick={withLabels ? { fill: COLORS.brandDark, strokeWidth: 0 } : false}
         type="number"
         tickMargin={tickMargin}
+        tickFormatter={tick => {
+          return tickMark ? intlNumberFormat(tick) : tick
```

Comment:
can't we handle this via i18next?

Reply:
I think we can handle this with bignumber. For now, let's just  use `number.toLocaleString()`, and add a TODO, and I will pulish it in #333.

Reply:
```
          tickFormatter={tick =>
            t('foo', {
              value: tick,
              formatParams: {
                value: {
                  notation: 'compact',
                } satisfies Intl.NumberFormatOptions,
              },
            })
          }
          ```
          in lang file `"foo": "{{value, number}}",`

Reply:
Right, thanks.  TypeScript was complaining for some reason, and I was thinking we need the polyfill. Removed.